lnrpc/walletrpc/walletkit_server.go

```diff
@@ -445,6 +449,26 @@ func (w *WalletKit) NextAddr(ctx context.Context,
 	}, nil
 }
 
+// ListWalletAddresses returns a list of all addresses known to the underlying wallet.
+func (w *WalletKit) ListWalletAddresses(ctx context.Context,
+	req *AddrRequest) (*AddrsResponse, error) {
+
+	var addrsSlice []string
```

Comment:
nit: move this below `ListWalletAddresses` and preallocate `addrsSlice := make([]string, 0, len(walletAddrs))`